assets/js/util.js

```diff
@@ -11,6 +11,8 @@ var TEXTAREA_AUTO_RESIZE_MINIMUM_WIDTH = 768,
     BACK_TO_TOP_BUTTON_ACTIVATION_HEIGHT = 300,
     APY_REQUEST_URL_THRESHOLD_LENGTH = 2000; // maintain 48 characters buffer for generated parameters
 
+var debounceDuration = 250;
```

Comment:
`debounceDuration` => `DEFAULT_DEBOUNCE_DURATION` and add to the constants above.